knowledge_base/python_wheel_poetry/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "python_wheel_poetry"
+version = "0.0.1"
+description = "Example package to demonstrate using Poetry with Databricks Asset Bundles."
+authors = ["Pieter Noordhuis <pieter.noordhuis@databricks.com>"]
+readme = "README.md"
+
+[tool.poetry.scripts]
+python_wheel_poetry = "python_wheel_poetry.hello:hello"
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
Can we have some external dependencies here and have an example to show how package all dependencies so they can be used later in jobs?

Reply:
I added `cowsay`.